src/main/scala/com/typesafe/sbt/packager/debian/NativePackaging.scala

```diff
@@ -43,6 +43,7 @@ trait DebianNativePackaging extends DebianPluginLike {
    * package.
    */
   private[debian] def debianNativeSettings: Seq[Setting[_]] = inConfig(Debian)(Seq(
+    debianNativeBuildOptions += "-Znone", // packages are largely JARs, which are already compressed
```

Comment:
We shouldn't scope this. The setting is already prefixed with _debian_. I just realised it after reading the tests.